modules/test/playwright/pages/layout-content-page-editor-web/PageEditorPage.ts

```diff
@@ -64,7 +64,7 @@ export class PageEditorPage {
 		await this.page.goto('/');
 
 		await this.page.goto(
-			`/web${siteUrl || '/guest'}${layout.friendlyUrlPath}?p_l_mode=edit${doAsUserId ? '&doAsUserId=' + doAsUserId : ''}`
+			`/web${siteUrl || '/guest'}${layout.friendlyUrlPath || layout.friendlyURL}?p_l_mode=edit${doAsUserId ? '&doAsUserId=' + doAsUserId : ''}`
```

Comment:
Inconsistency in what is used in `layout` is not good, but improving that is outside of scope of this task. This quick&dirty-ish fallback solution is fine by me :+1:

Reply:
I don't have objections either